src/Columns/ColumnAggregateFunction.h

```diff
@@ -221,6 +221,11 @@ class ColumnAggregateFunction final : public COWHelper<IColumnHelper<ColumnAggre
         return 0;
     }
 
+    bool equalsAt(size_t, size_t, const IColumn &) const override
+    {
+        return true;
```

Comment:
?

Reply:
It's consistent with `compareAt` which always returns 0 for `ColumnAggregateFunction`.